scripts/in_container/check_environment.sh

```diff
@@ -31,13 +31,21 @@ EXIT_CODE=0
 
 . "$( dirname "${BASH_SOURCE[0]}" )/check_connectivity.sh"
 
+export COLOR_YELLOW=$'\e[33m'
+export COLOR_RESET=$'\e[0m'
+
 function check_service {
     local label=$1
     local call=$2
     local max_check=${3:=1}
+    local sleep_time=${4:=0}
 
-   check_service_connection "${label}" "${call}" "${max_check}"
-   EXIT_CODE=$?
+    if [[ ${sleep_time} != 0 ]]; then
+        echo "${COLOR_YELLOW}Waiting ${sleep_time} seconds before checking ${label}${COLOR_RESET}"
```

Comment:
Not too obvious too the reader / debugger. Lets make it:
```suggestion
        echo "${COLOR_YELLOW}Adding initial delay: waiting ${sleep_time} seconds before checking ${label}.${COLOR_RESET}"

```

Reply:
Will handle in follow up